core/domain/user_services.py

```diff
@@ -21,7 +21,6 @@
 import datetime
 import logging
 import re
-
 from core.platform import models
```

Comment:
Please preserve the newline above this one, in order to separate Python standard libs from the others.